### PR TITLE
Upgrade Rails to 6.0.3.5

### PR DESCRIPTION
**Pull request by emerencia**

### CODE REVIEW

File: config/initializers/lograge.rb [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
+  # In some test specs, where we test JWT login without specifying a token,
+  # current_user raises an error, hence this fallback.
   config.lograge.custom_payload do |controller|
-    {
-      ip: controller.request.ip,
-      user_id: controller.current_user.try(:id)
-    }
+    begin
+      {
+        ip: controller.request.ip,
```
Comment (frbl) [inline config/initializers/lograge.rb:10]: do we really need to log IPs? Id rather not tbh

**Review by frbl — APPROVED** [review]
lgtm

File: config/initializers/lograge.rb (hunk 1 above)
Comment (emerencia) [reply config/initializers/lograge.rb:10]: Ok, I'll remove it.

Comment (emerencia) [reply config/initializers/lograge.rb:10]: Fixed.

**Pull request merged**